packages/arb-avm-cpp/tests/arbos.cpp

```diff
@@ -61,8 +61,7 @@ TEST_CASE("ARBOS test vectors") {
             storage.initialize(arb_os_path);
             auto mach = storage.getInitialMachine(value_cache);
             mach->machine_state.stack.push(uint256_t{0});
-            auto assertion =
-                mach->run(1000000000, messages, std::chrono::seconds{0});
+            auto assertion = mach->run(0, false, messages, nonstd::nullopt);
```

Comment:
We want to leave this as a large number rather than a 0 since we want execution to terminate if there's an issue that leads to an infinite loop rather than hanging